### PR TITLE
count for sent and pending notifications added

### PR DESCRIPTION
**Pull request by btensay**
Total counts for notifications with sent and pending status are added.
Test cases updated too.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java [hunk 1]
```diff
@@ -116,7 +117,21 @@ private NotificationsResponse mapToNotificationsResponse(List<Notification> list
...
             .collect(toList());
-        return new NotificationsResponse(notifications.size(), notifications);
+
+        var sentNotificationsCount = notifications.stream()
+            .filter(n -> n.status.equals("SENT"))
+            .collect(Collectors.toList()).size();
+
+        var pendingNotificationsCount = notifications.stream()
+            .filter(n -> n.status.equals("PENDING"))
+            .collect(Collectors.toList()).size();
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java:127]: in one loop ? like you did last time.

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java:127]: and this code can  move to 'NotificationsResponse'
and  "SENT" and "PENDING" can be constants

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java:127]: use PENDING.name(), SENT.name()

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java:127]: And previously counts were derived in the constructor of the response class

**Review by mdayican — APPROVED** [review]

**Review by apereverzin — APPROVED** [review]

**Pull request merged**